install.sh

```diff
@@ -34,9 +34,9 @@ mysql -u$user -e "create database librehatti;" -p$password
 
 cd src/librehatti/
 
-sed -i -e "s/'NAME': 'db_name'/'NAME': 'librehatti'/g" settings.py
```

Comment:
No, need to change `install.sh`. You may change `settings.py` and set `'USER'` to `db_user` instead of `root` and same for `NAME` and `PASSWORD`.